Polyhedron/demo/Polyhedron/Scene_polylines_item.h

```diff
@@ -104,7 +102,6 @@ public Q_SLOTS:
     mutable std::size_t nb_wire;
     mutable std::size_t nb_centers;
     mutable std::size_t nb_lines;
-    mutable QOpenGLShaderProgram *program;
     mutable   GLuint nbSpheres;
```

Comment:
@maxGimeno: `program` is removed. And the unused typedef `Iso_cuboid_3` as well.